airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/s3/S3StreamCopier.java

```diff
@@ -127,6 +127,19 @@ public void write(final UUID id, final AirbyteRecordMessage recordMessage, final
     }
   }
 
+  @Override
+  public void closeStagingFileWriter() throws Exception {
+    Set<String> removedKeys = new HashSet<>();
+    for (String key : activeStagingWriterFileNames) {
+      if (!key.equals(currentFile)) {
+        stagingWritersByFile.get(key).close(false);
+        stagingWritersByFile.remove(key);
+        removedKeys.add(key);
```

Comment:
Perhaps I'll leave it as it is, because. it is possible that several writers will be closed. If such a situation arises, then we will get a problem when deleting a single file name.